pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [project]
 # https://packaging.python.org/en/latest/specifications/declaring-project-metadata/
 name = "python-libs"
-version = "1.0.0"
```

Comment:
I can't recommend enough disabling hardcoded versions and using setuptools_scm to generate the version from git.

Should be as easy as
```
# pyproject.toml
[project]
# version = "0.0.1"  # Remove any existing version parameter.
dynamic = ["version"]

[build-system]
requires = ["setuptools >= 38.6.0", "wheel", "setuptools_scm[toml]"]

[tool.setuptools_scm]
```

Reply:
Done in PR 27!